apps/aehttp/test/aehttp_ws_test_utils.erl

```diff
@@ -0,0 +1,155 @@
+-module(aehttp_ws_test_utils).
+-behaviour(websocket_client).
+
+%% API
+-export([start/2,
+         send/4,
+         subscribe_for_event/3,
+         unsubscribe_for_event/3,
+         wait_for_event/2,
+         wait_for_event/3,
+         stop/1]).
+
+%% behaviour exports
+-export([
+         init/1,
+         onconnect/2,
+         ondisconnect/2,
+         websocket_handle/3,
+         websocket_info/3,
+         websocket_terminate/3
+        ]).
+
+start(Host, Port) ->
+    crypto:start(),
+    ssl:start(),
+    WsAddress = "ws://" ++ Host ++ ":" ++ integer_to_list(Port) ++ "/websocket",
+    ct:log("connecting to ~p", [WsAddress]),
+    {ok, Pid} = websocket_client:start_link(WsAddress, ?MODULE, self()),
```

Comment:
It does not matter much, though websocket_client:start_link types 3rd arg as list.